tools/class-parse/class-parse.csproj

```diff
@@ -12,11 +12,12 @@
     <ProductVersion>8.0.30703</ProductVersion>
     <SchemaVersion>2.0</SchemaVersion>
   </PropertyGroup>
+  <Import Project="..\..\Configuration.props" />
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\bin\Debug</OutputPath>
+    <OutputPath>$(UtilityOutputPath)</OutputPath>
```

Comment:
So, `$(UtilityOutputPath)` needs to be absolute, since it is being used by multiple projects. Or will this be set individually for each of these project builds? If it does need to be absolute then the name should reflect that.